common/fs/src/cache/tailed_file.rs

```diff
@@ -1,109 +1,233 @@
+#![allow(clippy::await_holding_refcell_ref)]
```

Comment:
Clippy doesn't like us holding onto refcell borrows across await points. Generally this is a good lint, especially if you're expecting to access the things inside the RefCell concurrently. 

Unfortunately we're storing the filehandle inside the refcell, so we need to hold it. However, we aren't accessing the file handle concurrently, we're only using the RefCell for interior mutability, only the Lines stream should access it and only one of those should exist at once.